Android/build.gradle

```diff
@@ -63,7 +63,7 @@ android {
     defaultConfig {
         applicationId 'org.droidplanner.android'
         minSdkVersion 14
-        targetSdkVersion 22
+        targetSdkVersion 23
```

Comment:
@jandrop let's leave it at `targetSdkVersion 22`, otherwise the app would need to be tested and add support for the new [Android M runtime permissions](https://developer.android.com/training/permissions/requesting.html). A task which is scheduled for the `3.3.x` release.